src/sapi.rs

```diff
@@ -294,7 +302,9 @@ impl SpVoice {
     }
 
     fn set_voice(&mut self, token: Speech::ISpObjectToken) {
-        unsafe { self.voice.SetVoice(&token).expect("set voice") }
+        unsafe {
+            self.voice.SetVoice(&token).ok();
```

Comment:
Why is this change necessary?

Reply:
It's not strictly required -- I changed it when I found some voices in the registry that crashed rust-reader when selected. This line wasn't the culprit, but I figured that failure to change the voice should not be a fatal error. Do you agree?

Reply:
Seems like a reasonable change! Although we should probably be consistent about whether `set_` error are fatal. An alternative is for all of the `set_` to return a Result, and let the caller figure out what to do with the error. (As I said, error handling in this code base isn't great.)

Reply:
Edit: this looks good, sorry. It's only used a couple of lines further down, and we handle the error by returning the name of the voice that is currently set.